test/distributed/_composable/test_fully_shard.py

```diff
@@ -8,9 +8,9 @@
 import torch
 import torch.distributed as dist
 import torch.nn as nn
+from torch.distributed._composable.fully_shard import fully_shard
```

Comment:
expose this API in `__init__.py` and import from `torch.distributed._composable`?

Reply:
Will do!